### ISSUE TITLE
c82d5d6 reverted the config restore: config_v7.json is still the widened+BN arm, and GRIT inherits BatchNorm on RRWP

### ISSUE THREAD

**Issue by kuanlinhuang** [Huang-lab/TRACE-Kin#12]
`c82d5d6` reverted the config restore 12 minutes after it landed, with the default revert message and no stated reason. The two commits cancel exactly:

```
$ git diff --stat 127bac6^ c82d5d6
(empty)
```

`127bac6`'s message is one of the most useful in the repo - it names the three keys `1d24a70` moved at once, says why that makes `config_v7.json` stop being the baseline, and separates the GRIT inheritance from the PNA change. If the revert was deliberate, that reasoning is now unrecorded and the next person re-derives it. If it was a mistake, four things are currently true that the message says should not be.

I am not assuming which. But the four consequences are worth stating because three of them affect a run in progress.

### 1. `config_v7.json` is the widened + BatchNorm arm, not the baseline

```
params.mol_pe_mode      = 'lspe'
params.mol_pe_steps     = 16      # was 8 before 1d24a70
params.mol_pe_dim       = 64      # was 16
params.mol_pe_raw_norm  = 'batch' # was absent
```

`net_v7`'s PNA path is byte-identical from `1d24a70` to `HEAD`, so a run after 2026-09-03 that records "config_v7.json" is not comparable to one before it, and nothing in the artifact says so. This is the part that silently invalidates a comparison rather than just costing time.

### 2. `config_v7_grit.json` applies BatchNorm to RRWP, and it is live

`mol_pe_raw_norm` reaches GRIT - `net_v7.py:609` passes it into `GRITLigandEncoder`, and `grit.py:268-272` builds two BatchNorms from it:

```python
self.pe_norm   = nn.BatchNorm1d(pe_steps - 2, eps=1e-3)   # RRWP diagonal
self.pair_norm = nn.BatchNorm1d(pe_steps, eps=1e-3)       # full pair RRWP
```

Executed at the `config_v7_grit.json` operating point (d_model 512, grit_dim 64, 10 layers, ksteps 21), identical seed and init, dropout off, `train()`:

```
max|none - batch| = 1.3746
relative          = 48.0% of ||atom_h||
```

So it is not a marginal knob - it is 48% of the ligand embedding.

The reference does not do this. In `LiamMa/GRIT`, both `RRWPLinearNodeEncoder` and `RRWPLinearEdgeEncoder` default to `batchnorm=False`, and `zinc-GRIT-RRWP.yaml` - the operating point `12b958c` matched everything else to - sets no norm at all:

```yaml
posenc_RRWP:
  enable: True
  ksteps: 21
  add_identity: True
```

The pair encoder additionally warns against it in the source, at `rrwp_encoder.py:116` and again at runtime:

```python
# note: batchnorm/layernorm might ruin some properties of pe on providing shortest-path distance info
warnings.warn("batchnorm/layernorm might ruin some properties of pe ...")
```

The mechanism is visible in the values: RRWP's zeros are informative - `[(D^-1 A)^k]_ij = 0` means "no walk of length k from i to j", which is how hop distance is encoded. After BatchNorm the "no walk" entries become a batch-dependent constant:

```
ch 0 (identity): 85.1% of pairs are 0  ->  all become -0.4175
ch 1 (k=1 bond): 70.1% of pairs are 0  ->  all become -0.6467
```

Worth being precise about the severity: `pair_pe_in` is `nn.Linear(pe_steps, e_dim)` **with** a bias here, where the reference's is `bias=use_bias` defaulting to False, so the shift is absorbable and this is an optimisation difference rather than a loss of expressivity. The problem is that it is not a chosen difference - the key was inherited by copying `config_v7.json`, where it means something else and is defensible (BatchNorm on RWSE is what GraphGPS does). A PNA-vs-GRIT comparison should differ in the layer stack, not in the layer stack plus an unintended normalisation of the positional encoding.

### 3. `config_v7_pe_wide_bn.json` is gone

`127bac6` created it precisely so the Sep-3 arm stayed runnable and attributable after `config_v7.json` went back to baseline. The revert deleted it, so that arm currently has no name.

### 4. The GRIT config misreports its own walk length

`mol_pe_steps: 16`, `mol_pe_dim: 64` and `mol_pe_mode: "lspe"` are all dead in the GRIT path - `net_v7.py:604-609` passes only `mol_grit_*` plus `mol_pe_raw_norm`, and `GRITLigandEncoder` takes no `pe_mode` (it returns `p=None`, so LSPE is off regardless). The walk length actually used is `mol_grit_steps: 21`, sitting two lines below `mol_pe_steps: 16`. A config that records the run should not name a number the run does not use - that is what `127bac6` dropped them for.

### What would settle it

Either re-apply `127bac6`, or keep the revert and say in a commit message that `config_v7.json` is now deliberately the widened+BN configuration and the pre-09-03 numbers are superseded. Both are defensible. The current state - where the file changed, the code did not, and nothing records it - is the one that is not.

Separately, `mol_pe_raw_norm` in `config_v7_grit.json` looks worth setting to `"none"` on its own merits regardless of how the rest is resolved, so the GRIT arm starts from the published operating point.

### Reproducing

```bash
git diff --stat 127bac6^ c82d5d6                     # empty
python -c "import json;print(json.load(open('training/config_v7_grit.json'))['params']['mol_pe_raw_norm'])"
sed -n '602,619p' models/trace_kin/net_v7.py         # the dispatch
sed -n '266,278p' models/trace_kin/grit.py           # the two BatchNorms
curl -s https://raw.githubusercontent.com/LiamMa/GRIT/main/grit/encoder/rrwp_encoder.py | sed -n '110,126p'
```

The 48% figure: instantiate `GRITLigandEncoder` twice from the same seed with `pe_raw_norm="none"` and `"batch"`, `train()` mode, and compare `atom_h` on the same input.
